mesheryctl/internal/cli/root/environments/list.go

```diff
@@ -18,72 +18,87 @@ import (
 	"fmt"
 
 	"github.com/google/uuid"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
 	"github.com/meshery/meshery/mesheryctl/internal/cli/pkg/api"
 	"github.com/meshery/meshery/mesheryctl/internal/cli/pkg/display"
 	"github.com/meshery/meshery/mesheryctl/pkg/utils"
 	"github.com/meshery/meshery/server/models/environments"
-	"github.com/pkg/errors"
-
-	"github.com/spf13/cobra"
 )
 
 var listEnvironmentCmd = &cobra.Command{
 	Use:   "list",
 	Short: "List registered environments",
-	Long: `List name of all registered environments
-Documentation for environment can be found at https://docs.meshery.io/reference/mesheryctl/environment/list`,
+	Long: `List the names of all registered environments.
+
+Documentation:
+https://docs.meshery.io/reference/mesheryctl/environment/list`,
 	Example: `
-// List all registered environment
-mesheryctl environment list --orgID [orgID]
+// List all registered environments for an organization
+mesheryctl environment list --orgID <org-id>
```

Comment:
```suggestion
mesheryctl environment list --orgID [org-id]
```
there is automation to generate the ClI reference page documentation and `<` or `>` had side effects in the past. Also `[]` are used everywhere else in the examples